packages/storybook/src/nimble/radio-group/radio-group.stories.ts

```diff
@@ -105,6 +114,16 @@ export const radioGroup: StoryObj<RadioGroupArgs> = {
                 'Event emitted when the user selects a new value in the radio group.',
             table: { category: apiCategory.events },
             control: false
+        },
+        errorText: {
+            name: 'error-text',
+            description: errorTextDescription,
+            table: { category: apiCategory.attributes }
+        },
+        errorVisible: {
+            name: 'error-visible',
+            description: errorVisibleDescription,
```

Comment:
`errorVisibleDescription` doesn't say anything about the exclamation point visual, but I think it should.